draft-ietf-ohai-ohttp.md

```diff
@@ -1124,7 +1137,9 @@ attacks; see {{ta}}.
 
 If separate entities provide the Oblivious Gateway Resource and Target Resource,
 these entities might need an arrangement similar to that between server and
-relay for managing denial of service; see {{dos}}.
+relay for managing denial of service; see {{dos}}. Moreover, the Oblivious Gateway
+Resource SHOULD apply some form of allowlist to ensure that the Oblivious Gateway
```

Comment:
This requirement should be sharper, say:

> MUST maintain a allow list of target resources that the Gateway will respond for, and check requests against this allow list

Reply:
I think SHOULD is fine for this case since there are alternative ways to solve this problem, and we have a MUST check on the client-side now. Resolving as such.